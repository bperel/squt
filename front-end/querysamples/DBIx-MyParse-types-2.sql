SELECT 1, TRUE
