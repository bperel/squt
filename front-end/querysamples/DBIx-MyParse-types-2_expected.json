{
	"Constants" : {
		"0" : {
			"to" : "OUTPUT",
			"alias" : "1",
			"value" : "1"
		},
		"1" : {
			"value" : "1",
			"to" : "OUTPUT",
			"alias" : "1"
		}
	}
}
